### PR TITLE
CAT-586 Add other categories to the cat-tool service

### PR DESCRIPTION
**Pull request by steverendell**
Add extra fields that may be set by create / approve categorisation

### CODE REVIEW

File: src/test/resources/features/booking_details.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@sdar
```
Comment (brightonsbox) [inline src/test/resources/features/booking_details.feature:1]: What is this doing?

File: src/main/java/net/syscon/elite/service/InmateService.java [hunk 2]
```diff
@@ -593,6 +608,13 @@ private void validate(final CategoryApprovalDetail detail) {
         } catch (final EntityNotFoundException ex) {
             throw new HttpClientErrorException(HttpStatus.BAD_REQUEST, "Committee Code not recognised.");
         }
+        if (StringUtils.isNotBlank(detail.getApprovedPlacementAgencyId())) {
+            try {
+                agencyService.getAgency(detail.getApprovedPlacementAgencyId(), ACTIVE_ONLY, "INST");
+            } catch (final EntityNotFoundException ex) {
+                throw new HttpClientErrorException(HttpStatus.BAD_REQUEST, "Review placement agency id not recognised.");
```
Comment (brightonsbox) [inline src/main/java/net/syscon/elite/service/InmateService.java:615]: Maybe could exercise these error conditions in tests?

File: src/test/resources/features/booking_details.feature (hunk 1 above)
Comment (steverendell) [reply src/test/resources/features/booking_details.feature:1]: whoops good spot

File: src/main/java/net/syscon/elite/service/InmateService.java (hunk 2 above)
Comment (steverendell) [reply src/main/java/net/syscon/elite/service/InmateService.java:615]: its covered in OffenderAssessmentResourceTest

**Pull request merged**